.github/workflows/test_iasworld_data.yaml

```diff
@@ -0,0 +1,141 @@
+name: test-iasworld-data
+
+# This workflow is not configured to run on PRs, because PRs have their own CI
+# process defined by the `build-and-test-dbt` workflow. Instead, the Data
+# Department's Spark data extraction process dispatches it via API once per
+# day after finishing the daily ingest of iasWorld data. For more detail, see:
+#
+# https://github.com/ccao-data/service-spark-iasworld
```

Comment:
This comment is slightly different, to reflect the new conditions in which this workflow runs compared to the old `test-dbt-models` workflow.